riker-macros/Cargo.toml

```diff
@@ -14,9 +14,9 @@ keywords = ["actors", "actor-model", "async", "cqrs", "event_sourcing"]
 proc-macro = true
 
 [dependencies]
-syn = { version ="0.15.39", features = ["parsing", "full", "extra-traits", "proc-macro"] }
-quote = "0.6.3"
-proc-macro2 = "0.4.4"
+syn = { version ="1.0", features = ["parsing", "full", "extra-traits", "proc-macro"] }
```

Comment:
There is already a pull request which fixes this. It shouldn't really matter, but I think it would be better to keep them separate.

Reply:
Sure - I can revert that change.

Reply:
I must have forgotten that I did that so late at night. I guess we can consider this as proof that the PR will also work with the updated `riker-macros` as proposed in #136